ui-src/src/drawing/drawGoalCard.js

```diff
@@ -29,6 +34,29 @@ function roundRect(ctx, x, y, w, h, radius, color, stroke, strokeWidth) {
   else ctx.fill()
 }
 
+// line wrapping code from https://stackoverflow.com/questions/2936112/
+function getLines(ctx, text, maxWidth) {
+  var words = text.split(' ')
```

Comment:
can you convert `var` to `const` or `let` depending on what's right

Reply:
Done!